scripts/build_pypi_package.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+script_dir=`dirname $0`
+cd ${script_dir}/..
+rm -rf ./slack_bolt.egg-info
```

Comment:
Shouldn't it be `slack_sdk`?
```suggestion
rm -rf ./slack_sdk.egg-info
```